docs/content/latest/explore/ysql-language-features/advanced-features/partitions.md

```diff
@@ -2,13 +2,14 @@
 title: Table Partitioning
 linkTitle: Table Partitioning
 description: Table Partitioning in YSQL
-headcontent: Table Partitioning in YSQL
 image: /images/section_icons/secure/create-roles.png
 menu:
   latest:
-    identifier: explore-ysql-language-features-partitioins
-    parent: explore-ysql-language-features
-    weight: 315
+    identifier: advanced-features-partitioins
```

Comment:
Search for any other occurrences of the typo!
```suggestion
    identifier: advanced-features-partitions
```

Reply:
Thank you! Yes , found in its stable version as well. Will make the change there too.